java-api-core/src/com/kinvey/java/model/KinveySyncSaveBatchResponse.kt

```diff
@@ -1,5 +1,6 @@
 package com.kinvey.java.model
 
-class KinveySyncSaveBatchResponse<T>(
+data class KinveySyncSaveBatchResponse<T>(
```

Comment:
sorry about my lack of better knowledge about Kotlin, but what's the difference between a regular `class` and `data class`?

Reply:
Data classes are designed to store data, can be useful for network request entities.
This type of classes can automatically create equals and hashCode implementation with variables that in class constructor.
If we use class for store or transfer data then preferable to use data classes instead of regular classes.
So I used `data class` in such cases.